numba_dpex/core/pipelines/kernel_compiler.py

```diff
@@ -4,10 +4,9 @@
 
 from numba.core.compiler import CompilerBase
 from numba.core.compiler_machinery import PassManager
-from numba.core.typed_passes import (
+from numba.core.typed_passes import (  # We are using FreevarDisambiguationLowering instead; numba-dpex github issue: 898; NativeLowering,
```

Comment:
Comment not needed here. You can add the reference to the issue with URL to the docstring of `QualNameDisambiguationLowering` class.

Reply:
Fixed.